website/docs/cli/configuration/terminal.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Terminal Settings
+sidebar_position: 4
+description: Configure terminal output settings including syntax highlighting
+---
+
+import Intro from '@site/src/components/Intro';
+import KeyPoints from '@site/src/components/KeyPoints';
+import Note from '@site/src/components/Note';
+import File from '@site/src/components/File';
+
+<Intro>
+Atmos provides configurable terminal settings that allow you to customize the output appearance, including syntax highlighting for YAML and JSON outputs. These settings can be configured in your `atmos.yaml` configuration file.
+</Intro>
+<KeyPoints>
+- Configure syntax highlighting for terminal output
+- Customize color schemes and formatting options
+- Control output pagination and line wrapping
+- Set display preferences for different output formats
+</KeyPoints>
+## General Terminal Settings
+
+Configure general terminal behavior:
+
+```yaml
+settings:
+  terminal:
+    max_width: 120          # Maximum width for terminal output
+    use_pager: true         # Whether to use a pager (like 'less') for long output
+    timestamps: false       # Show timestamps in output
+    colors: true           # Enable colored output
+    unicode: true          # Use Unicode characters in output
+```
+
+## Syntax Highlighting
+
+You can customize the syntax highlighting behavior for terminal output using the following settings:
+
+```yaml
+settings:
+  terminal:
+    syntax_highlighting:
+      enabled: true         # Enable/disable syntax highlighting
+      lexer: yaml           # Lexer to use (yaml or json)
+      formatter: terminal   # Output formatter
+      style: dracula        # Color scheme to use
+      use_pager: false      # Whether to use a pager for highlighted output
+      options:
+        line_numbers: false # Show line numbers
+        wrap: false         # Wrap long lines
+```
+
+### Configuration Options
+
+<dl>
+  <dt>`enabled`</dt>
+  <dd>Enable or disable syntax highlighting (default: `true`)</dd>
+  
+  <dt>`lexer`</dt>
+  <dd>
+    Specify the lexer to use for syntax highlighting:
+    <dl>
+      <dt>`yaml`</dt>
+      <dd>For YAML output</dd>
+      <dt>`json`</dt>
+      <dd>For JSON output</dd>
+    </dl>
+  </dd>
+  
+  <dt>`formatter`</dt>
+  <dd>Output formatter (default: `terminal`)</dd>
+  
+  <dt>`style`</dt>
```

Comment:
When did we switch to calling it `style` as opposed to `theme`? I prefer we keep `theme`